src/common/utilities/color-utils.ts

```diff
@@ -8,4 +8,14 @@ export const colors = {
   green: "#41bc7a",
   yellow: "#f1c232",
   red: "#ff6961",
+  white: "#FFFFFF"
 };
+
+export const heatMapColors = {
+  greenLight: "#9af4a0",
+  greenMediumLight: "#66d979",
+  greenMedium: "#41bc7a",
+  greenMediumDark: "#339a55",
+  greenDark: "#1e7f39",
+  greenDarkest: "#005600",
```

Comment:
since they are all green, you can remove the word green and maybe call the main variable: `greenHeatMapShades`